### DISCUSSION TITLE
Installed  PI4J on Ubuntu Server 20

### DISCUSSION THREAD

**Discussion by raydacosta** [Pi4J/pi4j#347] [General]
How should I proceed to run PI4J on Ubuntu Server 20 installed on Rraspberry 3 B+

**Comment by taartspi** [comment]
I only use the raspberry OS.  But looking at the pi4j web site you are directed to https://github.com/thibmaek/awesome-raspberry-pi/blob/master/README.md.    I do not see an ubuntu server for the pi4.   Possibly that site where you were directed has a forum to ask about your OS usage
